create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -93,6 +101,17 @@ void shouldExtractDoiAndPlaceItInsideReferenceObject() throws IOException {
         assertThat(createPublicationRequest.getEntityDescription().getReference().getDoi(), equalToObject(expectedURI));
     }
 
+    @Test
+    void shouldExtractMainTitleAndPlaceItInsideEntityDescriptionObject() throws IOException {
```

Comment:
shouldReturnCreatePublicationRequestWithMainTitle is good enough for me, but opinion may vary :D

Reply:
Fixed